benchmarking/daint_single_node/run_benchmark.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python3
+
+# This script generates a rundirectory given user-provided arguments and a configuration
+# file. It relies on fv3config for setting up the run directory. Once done, a SLURM
+# script is generated and submitted. Optionally, we wait for the completion of the SLURM
+# job.
+
+# 2021/01/22 Oliver Fuhrer, Vulcan Inc, oliverf@vulcan.com
+
+import math
+import os
+import sys
+import shutil
+import hashlib
+import yaml
+import click
+import fv3config
+
+# constants
+DAINT_SECTION = 'hybrid' # one of hybrid or multicore
+FV3_NUMBER_OF_TILES = 6
+
+# Piz Daint specification
+if DAINT_SECTION == 'hybrid':
+    SOCKETS_PER_NODE = 1
+    CORES_PER_SOCKET = 12
+    HAS_HYPERTHREADING = True
+if DAINT_SECTION == 'multicore':
+    SOCKETS_PER_NODE = 2
+    CORES_PER_SOCKET = 18
+    HAS_HYPERTHREADING = True
+
+@click.command()
+@click.option('--blocksize', default=-1, help='block size to use for physics')
+@click.option('--executable', default='./fv3.exe', help='path to the FV3GFS executable')
+@click.option('--force/--no-force', default=False, help='overwrite existing run directory (if it exists)')
+@click.option('--hyperthreading/--no-hyperthreading', default=False, help='switch hyperthreading on or off')
+@click.option('--module_env', default=None, help='file to source to pre-load modules')
+@click.option('--nodes_per_tile_side', default=1, help='number of compute nodes per tile side')
+@click.option('--partition', default='normal', help='parittion to submit job to')
+@click.option('--rank_layout', default=-1, help='Choose which rank layout to use (1, 2, 3, ...)')
+@click.option('--threads_per_rank', default=1, help='number of OpenMP threads per rank')
+@click.option('--timesteps', default=61, help='number of timesteps')
```

Comment:
can we default to 60?
```suggestion
@click.option('--timesteps', default=60, help='number of timesteps')
```

Reply:
Oh, interesting. Since I am throwing away the timing of the first timestep, I set this to 61 in order to have the 60 timesteps that we actually want to time. How are you doing that?

Reply:
I do 59 + 1